chat/src/components/message/BotErrorMessage.tsx

```diff
@@ -0,0 +1,66 @@
+import { MessageHeader } from "./MessageHeader";
+import { MessageContainer } from "./MessageContainer";
+import styled from "styled-components";
+import { MessageContextProvider } from "../../hooks/useMessageContext";
+import { CodeButton } from "../general/CodeButton";
+import { ReactComponent as RegenerateIcon } from "../../assets/regenerate-icon.svg";
+import { useConversationContext } from "../../hooks/useConversationContext";
+
+type Props = {
+  onRegenerate(): void;
+};
+
+export function BotErrorMessage({ onRegenerate }: Props): React.ReactElement {
+  const { id: conversationId } = useConversationContext();
+  return (
+    <MessageContextProvider
+      isError
+      message={{
+        conversationId,
+        isBot: true,
+        text:
```

Comment:
Link to support page?

Reply:
this is just a message for the analytics, but I'll add it.